Website/plugins/search-bar/add-search.raku

```diff
@@ -30,36 +29,35 @@ sub ( $pp, %processed, %options ) {
         $s.trans([</ \\ ">] => [<\\/ \\\\ \\">]);
     }
     sub escape-json(Str $s) is export {
-        $s.subst(｢\｣, ｢%5c｣, :g).subst('"', '\"', :g).subst(｢?｣, ｢%3F｣, :g)
-    }
-    for %defns.kv -> $fn, %targets {
-        for %targets.kv -> $targ, %info {
-            my $category = %info<category>.tc ;
-            $category = %info<subkind>.tc ~ ' operator' if $category eq 'Operator';
-            $categories{ $category }++;
-            @entries.push: %(
-                :$category,
-                :value( escape( %info<name> ) ),
-                :info( ': in <b>' ~ escape-json($fn) ~ '</b>' ),
-                :url( escape-json( "/$fn\#$targ" ) )
-            )
-        }
+        $s.subst(｢\｣, ｢%5C｣, :g).subst('"', '\"', :g).subst(｢?｣, ｢%3F｣, :g)
     }
+    #    for %defns.kv -> $fn, %targets {
+    #        for %targets.kv -> $targ, %info {
+    #            my $category = %info<category>.tc ;
+    #            $category = %info<subkind>.tc ~ ' operator' if $category eq 'Operator';
+    #            $categories{ $category }++;
+    #            @entries.push: %(
+    #                :$category,
+    #                :value( escape( %info<name> ) ),
+    #                :info( ': in <b>' ~ escape-json($fn) ~ '</b>' ),
+    #                :url( escape-json( "/$fn\#$targ" ) )
+    #            )
+    #        }
+    #    }
```

Comment:
Is this only commented out code? If so, should remove it.

Reply:
I noticed that there were duplicates for some secondaries, eg. `%%`. This seemed to be the offending code. The duplicates have gone.
Logically, the commented out code can be safely deleted. However, I retained a copy of the code in case it turns out that more references were eliminated.

Reply:
deleting